### PR TITLE
Rich text pages

### LINKED ISSUE #112: Re-check all the fonts + font fallbacks

### PR DESCRIPTION
**Pull request by kailasnadh790**
## Ticket/Issue number(s)
Fix #112 

## Test URLs
- Before: https://main--888de--aemsites.hlx.page/sicherheit-datenschutz/betruegerische-emails
- After: https://bug-112--888de--aemsites.hlx.page/sicherheit-datenschutz/betruegerische-emails
- After: https://bug-112--888de--aemsites.hlx.page/sicherheit-datenschutz/cookie-unternehmenspolitik
- After: https://bug-112--888de--aemsites.hlx.page/sicherheit-datenschutz/keine-spyware
- After: https://bug-112--888de--aemsites.hlx.page/sicherheit-datenschutz/cookie-unternehmenspolitik
- After: https://bug-112--888de--aemsites.hlx.page/sicherheit-datenschutz/quelle-der-geldmittel

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -25,12 +25,8 @@
...
 @font-face {
   font-family: "medium888-Fallback";
-  src: local('Arial');
-  font-weight: 400;
-  ascent-override: 83.10%;
-  descent-override: 21.87%;
-  line-gap-override: 0.00%;
-  size-adjust: 109%;
+  size-adjust: 112.311%;
+  src: local("Georgia");
```
[the remark this answers is not in the document]
Comment (kailasnadh790) [reply styles/styles.css:29]: this is giving me 100 LHS on my devtools LHS...but not sure why the bot is failing.

**Comment by helms-charity** [comment]
@kailasnadh790 I just that you are adding an inline style on every element, I think this is not necessary. 
image

**Comment by helms-charity** [comment]
@kailasnadh790 I found a new way to compare fallback fonts with the custom font.
https://meowni.ca/font-style-matcher/
According to this, if I compare 888 medium size 16px with Arial, these attributes should make them really close?

font-family: Arial
font-size: 17px
line-height: 1.5
font-weight: 400
letter-spacing: 0.6px;

image

**Comment by kailasnadh790** [comment]
current state is better : https://github.com/aemsites/888de/pull/131

**Review by helms-charity — APPROVED** [review]
This looks good

**Pull request merged**